api/v1/helpers/verbs/postUtils.js

```diff
@@ -81,6 +82,14 @@ function handlePostResult(o, resultObj, props, res, req) {
       realtimeEvents.sample.add, props.associatedModels.aspect);
   }
 
+  // order collectors by name
+  if (props.modelName === 'Generator' && o.collectors) {
```

Comment:
I thought one of the purposes of the refactor, out, patch and post story was not to do something like this ''props.ModelName ==='Generator'  for a specific model. Wondering if you want to move it to the controller. Same comment for get, patch and put.

Reply:
added moving the generator specific part to the refactor responsify story. 
this will be done in a separate PR.